frontend/src/stores/calendar-store.ts

```diff
@@ -44,6 +52,48 @@ export const useCalendarStore = defineStore('calendars', () => {
     }
   };
 
+  /**
+   * Get all cremote events from connected calendars for given time span
+   * @param call preconfigured API fetch function
+   * @param activeDate Dayjs object defining the current month
+   * @param force force a refetch
+   */
+  const getRemoteEvents = async (call: Fetch, activeDate: Dayjs, force = false) => {
+    // Get month identifier to remember this month's events are already retrieved
+    const month = activeDate.format('YYYY-MM');
```

Comment:
I just kept our decision to retrieve events on a monthly basis. But if you wish I can make that configurable as well.

Reply:
Do we have a define for that date format?

Reply:
No, but now we have. Thanks for this suggestion!